source/extensions/health_checkers/mysql/mysql.cc

```diff
@@ -0,0 +1,323 @@
+#include "extensions/health_checkers/mysql/mysql.h"
```

Comment:
There is some missing coverage for error cases in this file. Can you check the coverage build in CI and add some more tests?

Reply:
will do

Reply:
@mattklein123 where can I find the coverage build in CI? I can't find the build checks. Maybe because the PR was closed?